mkt/tags/models.py

```diff
@@ -57,3 +56,23 @@ class AddonTag(ModelBase):
 
     class Meta:
         db_table = 'users_tags_addons'
+
+
+def attach_tags(objs, m2m_name=None):
+    """
+    Fetch tags from `objs` in one query and then attach them to a property on
+    each instance. The name of the property will be `m2m_name` + '_list'.
+
+    Assumes every instance in `objs` uses the same model, and needs `m2m_name`
+    to be set to the name of the m2m field between the model and Tag.
+    """
+    if m2m_name is None:
```

Comment:
But why ? It's annoying that we accept it as a positional argument but ultimately it's not such a big deal, since it works anyway. I made it a keyword argument because I preferred to stay as explicit as I could, but there is no need to enforce that the caller is always using it like I intended, IMHO.